exercises/two-fer/.meta/.version

```diff
@@ -0,0 +1 @@
+3
```

Comment:
Change the version back to 1.

To do this, locally change this number to `0` then re-generate the tests, which should update it to 1 everywhere.